server/.env.example

```
@@ -8,4 +9,6 @@ REFRESH_TOKEN_EXPIRY=864000
 CLOUDINARY_CLOUD_NAME=
 CLOUDINARY_API_KEY=
 CLOUDINARY_API_SECRET=
-DEVELOPEMENT_MODE="false"
\ No newline at end of file
+DEVELOPEMENT_MODE="false"
+GITHUB_CLIENT_ID=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The GITHUB_CLIENT_ID key should go before the MONGO_URL key

(UnorderedKey)